app/assets/javascripts/controllers/floating_ip/floating_ip_form_controller.js

```diff
@@ -51,6 +51,10 @@ ManageIQ.angular.app.controller('floatingIpFormController', ['$http', '$scope',
     $http.get('/floating_ip/networks_by_ems/' + id)
       .then(getNetworkByEmsFormData)
       .catch(miqService.handleFailure);
+
+    miqService.getProviderTenants(function(data) {
+      $scope.available_tenants = data.resources;
```

Comment:
Sorry, just merged a PR converting floating ip to `controllerAs`.. This will need to be `vm.available_tenants = data.resources` now..